test/unit_test.dart

```diff
@@ -11,26 +9,26 @@ void main() {
 
   test('subtract two numbers', () {
     final calculator = Calculator();
-    // TODO add expect
+    expect(calculator.subtract(5, 3), 2);
   });
 
   test('multiply two numbers', () {
     final calculator = Calculator();
-    // TODO add expect
+    expect(calculator.multiply(4, 3), 12);
   });
 
   test('divide two numbers', () {
     final calculator = Calculator();
-    // TODO add expect
+    expect(calculator.divide(6, 3), 2);
   });
 
   test('mod two numbers', () {
     final calculator = Calculator();
-    // TODO add expect
+    expect(calculator.mod(7, 3), 1);
   });
 
   test('power two numbers', () {
     final calculator = Calculator();
-    // TODO add expect
+    expect(calculator.power(2, 3), 8);
```

Comment:
You did not added it in Calculator class

Reply:
sorry, what did i not add?  Power function is in calculator.dart

Reply:
you are right

Reply:
missed it

Reply:
Updated +1 point